### PR TITLE
Add a bio for @saimed

### PR DESCRIPTION
**Pull request by saimed**
This pull request adds a bio for @saimed as requested #22 

:+1: 
:star:

### CODE REVIEW

File: docs/saimed.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GitHub Professor: saimed
+
+**Name: ** Github saimed
+
+**Occupation: ** Dev
+
+**Location: ** MKE
```
Comment (saimed) [inline docs/saimed.md]: :+1:

Comment (PaulSchemenauer) [inline docs/saimed.md]: Not everyone may know what MKE stands for

**Comment by saimed** [comment]
Good point :+1:

File: docs/saimed.md (hunk 1 above)
Comment (saimed) [inline docs/saimed.md]: Good point :+1:  :100:

**Comment by saimed** [comment]
:speedboat: it

**Pull request merged**